lib/configuration.yaml

```diff
@@ -180,6 +180,45 @@ bench: &bench
           protocolMagic: 55550007
 
 
+smallbench: &smallbench
+  <<: *dev
+
+  core:
+    <<: *dev_core
+
+    genesis:
+      spec:
+        <<: *dev_core_genesis_spec
+        initializer:
+          testBalance:
+            poors:        50000
+            richmen:      3
```

Comment:
we need a new configuration with exactly three "richmen" (those can create blocks)